content/docs/install/cli-install/index.md

```diff
@@ -4,7 +4,12 @@ description: Install binaries for the Keptn CLI
 weight: 35
 ---
 
-Every Keptn release provides binaries for the Keptn CLI. These binaries are available for Linux, macOS, and Windows.
+The Keptn CLI must be installed on the local machine.
+It is used to send commands to Keptn by interacting with the API of Keptn.
+To communicate with Keptn, you must know a shared secret
+that is generated during the installation and verified by the *api* component.
```

Comment:
I think we can assume that users know what an API token is. Maybe this could be re-written.

Reply:
I reworded this to just say that it is generated during installation and actually call it an API token rather than a "shared secret" which yeah, they should probably know that they are the same thing but it does require an extra mental hop. And I linked to the section that has all the gnarly details about managing the API token if they need it.  See what you think.

Reply:
sounds good now!